.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Linux Conda
+
+on:
+  pull_request:
+    branches:
+      - master
+      - dev
+  push:
+    branches: [ master ]
+
+jobs:
+  build-linux-conda:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python '3.10'
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+      - name: Add conda to system path
+        run: |
+          # $CONDA is an environment variable pointing to the root of the miniconda directory
+          echo $CONDA/bin >> $GITHUB_PATH
+
+      - name: Install mamba
+        run: |
+          conda install -c conda-forge mamba
+          conda clean -a
+          
+          
+          #   - name: install caiman
+          #     run: |
+          #       mamba install -c conda-forge caiman
+
+      - name: install caiman
+        run: |
+          mamba env update -n base --file environment.yml
+          mamba install Cython -y
```

Comment:
environment.yml covers the cython install

Reply:
Hmm I am getting 

```pytb
ERROR: Failure: ImportError (cannot import name 'oasis' from partially initialized module 'caiman.source_extraction.cnmf' (most likely due to a circular import) (/home/runner/work/CaImAn/CaImAn/caiman/source_extraction/cnmf/__init__.py))
```

which indicates that oasis isn't being compiled correctly on `pip install`. This is I think the last critical piece. Any tips?